network/wsPeer.go

```diff
@@ -446,6 +513,17 @@ func (wp *wsPeer) writeLoopSend(msg sendMessage) (exit bool) {
 		// just drop it, don't break the connection
 		return false
 	}
+	if msg.msgTags != nil {
```

Comment:
is this the case that (if msg.msgTags != nil), then the writeLoopSend call is to register the peer's interests, and not actually send some message to the peer? 
A little comment about this can be helpful here if this is the case.

Reply:
that is correct. I'll add a comment.